examples/utils/migration-tools/api-report/migration-tools.public.api.md

```diff
@@ -0,0 +1,9 @@
+## Public API Report File for "@fluid-example/migration-tools"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// (No @packageDocumentation comment for this package)
```

Comment:
Recommend adding `@packageDocumentation` block in index.ts to have something and make this go and stay away.